donut/modules/bodfeedback/routes.py

```diff
@@ -0,0 +1,168 @@
+import flask
+from flask import Markup
+from donut.modules.bodfeedback import blueprint
+from donut.modules.bodfeedback import helpers
+from donut import auth_utils
+from donut.modules.bodfeedback.permissions import bod_permissions
+from donut.default_permissions import Permissions as default_permissions
+
+
+@blueprint.route('/bodfeedback')
+def bodfeedback():
+    summary = False
+    if auth_utils.check_login():
+        perms = auth_utils.get_permissions(flask.session['username'])
+        if default_permissions.ADMIN in perms or bod_permissions.SUMMARY in perms:
+            summary = True
+    return flask.render_template('bodfeedback.html', summary=summary)
+
+
+# Submit feedback form
+@blueprint.route('/bodfeedback/submit', methods=['POST'])
+def bodfeedback_submit():
+    fields = ['name', 'email', 'subject', 'msg']
+    required = ['subject', 'msg']
+    data = {}
+    for field in fields:
+        data[field] = flask.request.form.get(field)
+    for field in required:
+        if not data[field]:
+            flask.flash('Please fill in all required fields (marked with *)',
+                        'error')
+            return flask.redirect(flask.url_for('bodfeedback.bodfeedback'))
+    complaint_id = helpers.register_complaint(data)
+    flask.flash(
+        Markup('Success: <a href="' + helpers.get_link(complaint_id) +
+               '">View Complaint</a>'))
```

Comment:
Maybe add an extra note telling ghdm to save this url if they want to keep track of this complaint?